.drone.env

```diff
@@ -1,3 +1,3 @@
 # The version of OCIS to use in pipelines that test against OCIS
-OCIS_COMMITID=55f3ef0aff4c2ee58e4cf7b8a38391b43c0c576f
-OCIS_BRANCH=master
+OCIS_COMMITID=0c68af5369aed5fe74a5952ed48a06e3cb335b17
+OCIS_BRANCH=fix-ocdav-spaceid
```

Comment:
to be reverted and updated after https://github.com/owncloud/ocis/pull/7975 is merged